### ISSUE TITLE
Example package for rez-test?

### ISSUE THREAD

**Issue by dbr** [nerdvegas/rez#958]
I had a look around and couldn't really find much info on how to set up a package to work with `rez test` (mostly just some discussion before the feature was implemented)

Only info I could find on even how to structure the `tests` attribute in `package.py` was a comment on #665 - with a bit of tweaking that worked, but it would be nice if there was a bit more complete example documented somewhere :D

```
name = "test_example"
version = "0.1.0"

requires = [
    ...
]

def commands():
    env["PYTHONPATH"].append("{root}/python")


tests = {
    'unit': {
        'command': 'pytest {root}/python/',
        'requires': ['pytest'],
    }
}
```

**Comment by nerdvegas** [comment]
See https://github.com/nerdvegas/rez/wiki/Package-Definition-Guide#tests. I do think there should be a `Testing Packages` page however.

**Comment by dbr** [comment]
Ah, perfect, that page has all the info I need for now - thanks!

**Comment by nerdvegas** [comment]
Closed, see #996
